group_vars/mysql/public.yml

```diff
@@ -9,8 +9,16 @@ TARSNAP_ARCHIVE_NAME: "{{ MYSQL_SERVER_DOMAIN }}"
 mysql_root_password_update: true
 
 mysql_packages:
-  - mysql-server-5.5
-  - mysql-client-5.5
+  - mysql-server-5.6
+  - mysql-client-5.6
+
+mysql_server_id: "1"
+mysql_replication_role: 'master'
+mysql_replication_master: "{{ MYSQL_REPLICATION_MASTER_HOST }}"
+mysql_replication_user:
+    name: "{{ MYSQL_REPLICATION_USERNAME }}"
+    password: "{{ MYSQL_REPLICATION_PASSWORD }}"
+    host: "%"
```

Comment:
I had a quick look at the documentation of the mysql Ansible role we are using.  Do we also need to set `mysql_max_binlog_size`?  Apparently that's only the file size after which the binary log file gets rotated, and defaults to 1GB, but our role reduces it to 100MB.  Probably does not really matter, though.

Reply:
We could set this back to 1GB if 100MB seems too low. I'm not sure how much difference it will make either way though.
There could be a small performance impact to frequent file rotations I suppose? Otherwise I'm not sure how different it will be.